lib/passport.js

```diff
@@ -83,6 +84,11 @@ api.authenticate = function(strategy, options, callback) {
       // 3. The request method is in a list of default permitted CORS methods.
       // TODO: add an option to allow controlling permittedCorsMethods for
       // particular handlers that know what they're doing
+      if(!callback) {
```

Comment:
Place this conditional above the above comment section which pertains to the code below the conditional.